Dependency error after NodeJS update.

Hey guys, anyone know when html-injector will update dependencies? I start to get this error since i update node.

I really appreciate any kind of help on that issue:

npm install bs-html-injector --save-dev

> contextify@0.1.14 install /Users/Projects/app/node_modules/contextify
> node-gyp rebuild

14 errors generated.
make: *** [Release/obj.target/contextify/src/contextify.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)

gyp ERR! System Darwin 14.5.0
gyp ERR! command "/usr/local/Cellar/node/4.1.1/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/Projects/app/node_modules/contextify
gyp ERR! node -v v4.1.1
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok

Thanks!

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - '0.10'
+  - '4.2.0'

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "bs-html-injector",
-  "version": "2.0.4",
+  "version": "2.0.5",
   "description": "Inject only HTML that has changed.",
   "main": "index.js",
   "repository": {
@@ -17,17 +17,17 @@
   "license": "MIT",
   "devDependencies": {
     "browser-sync": "^2.7.6",
-    "chai": "^1.9.1",
-    "mocha": "^1.21.4",
-    "multiline": "^0.3.4",
+    "chai": "^3.3.0",
+    "mocha": "^2.3.3",
+    "multiline": "^1.0.2",
     "socket.io-client": "^1.1.0"
   },
   "dependencies": {
     "debug": "^2.1.3",
     "dom-compare-temp": "^0.1.0",
-    "jsdom": "^3.1.2",
+    "jsdom": "^6.5.1",
     "lodash": "^3.6.0",
-    "opt-merger": "^0.2.1",
+    "opt-merger": "^1.1.0",
     "request": "^2.40.0"
   },
   "keywords": [

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "bs-html-injector",
-  "version": "2.0.5",
+  "version": "3.0.0",
   "description": "Inject only HTML that has changed.",
   "main": "index.js",
   "repository": {

